snapshots/climate/2024-01-31/sea_surface_temperature_world.csv.dvc

```diff
@@ -0,0 +1,25 @@
+meta:
+  origin:
+    producer: Met Office Hadley Centre
+    title: HadSST
+    title_snapshot: HadSST - World
+    citation_full: 'Kennedy, J. J., Rayner, N. A., Atkinson, C. P., & Killick, R.
+      E. (2019). An ensemble data set of sea-surface temperature change from 1850:
+      the Met Office Hadley Centre HadSST.4.0.0.0 data set. Journal of Geophysical
+      Research: Atmospheres, 124. https://doi.org/10.1029/2018JD029867'
+    attribution: Met Office Hadley Centre - HadSST (2024)
+    attribution_short: Met Office
+    version_producer: 4.0.1.0
```

Comment:
True, that's because the paper describes version 4, and by now there is a new sub-version, 4.0.1.0, which we are using. I've changed the citation full a bit.